guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -1,32 +1,61 @@
 [id="Using_Container_Registries_{context}"]
 = Using container registries
 
-Podman and Docker can be used to fetch content from container registries.
+You can use Podman and Docker to fetch content from container registries and push the content to the {Project} container registry.
+The {Project} registry follows the Open Containers Initiative (OCI) specification, so you can push content to {Project} by using the same methods that apply to other registries.
+For more information about OCI, see link:https://opencontainers.org/[Open Container Initiative Distribution Specification].
+
+.Prerequisites
+* To push content to an organization or product, ensure your {Project} account has the `edit_products` permission.
+* To pull content, ensure that your {Project} account has the `view_lifecycle_environments`, `view_products`, and `view_content_views` permissions, unless the lifecycle environment allows unauthenticated pull.
```

Comment:
```suggestion
* To push content to {Project}, ensure your {Project} account has the `edit_products` permission.
* To pull content from {Project}, ensure that your {Project} account has the `view_lifecycle_environments`, `view_products`, and `view_content_views` permissions, unless the lifecycle environment allows unauthenticated pull.
```
But we say below that they have to specify both organization and product, so it isn't "or".
It might be a good idea to say from/to where the user is pulling/pushing - {Project} as opposed to other container registries instead.